features/items.md

```diff
@@ -0,0 +1,237 @@
+```
+
+To take this a step further you can begin to nest groups like in the example below:
+
+```xtend
+Group  All
+Group  gSensor             (All)
+Group  gTemperature        (gSensor)
+
+Number  Sensor_Temperature  "The Temperature is [%.1f °C]"  <temperature>  (gTemperature)  {knx="1/0/15+0/0/15"}
+```
+
+The item `Sensor_Temperature` is a member of the group `gTemperature`, which is itself a member of the group `gSensor`, which is a member of the group `All`.
+
+The item will only be included into each group once, regardless of the number of times the group is nested. e.g. The item `Sensor_Temperature` only exists once in the group `All`.
+
+Group items can also be used to easily determine one or more items with a defined value or can be used to calculate a value depending on all values within the group. Please note that this can only be used if all items in the group have the same type. The format for this is:
+
+```xtend
+Group:itemtype:function  itemname  ["labeltext"]  [<iconname>]  [(group1, group2, ...)]
+```
+
+By default, if no function is provided to the group, the Group uses OR. So for a Group of switches the Group state will be ON if any of the members states are ON. But this means that once one Item in the group has its state change to ON, the Group's state gets set. Each subsequent Item that changes state to ON will not trigger "myGroup changed" because the Group isn't changing.
+
+This is not a bug, it is the expected and designed behaviour. Because the group state is an aggregate, every change in the Item members does not necessarily result in a change to the Group's state.
+
+Group functions can be any of the following:
+
+Function | Description
+---------|------------
+AND(value1, value2) | This does a logical 'and' operation. Only if all items are of 'value1' this is returned, otherwise the 'value2' is returned.
+AVG                 | Calculates the numeric average over all item values of decimal type.
+MAX                 | This calculates the maximum value of all item values of decimal type.
+MIN                 | This calculates the minimum value of all item values of decimal type.
+NAND(value1, value2)| This does a logical 'nand' operation. The value is 'calculated' by the normal 'and' operation and than negated by returning the opposite value. E.g. when the 'and' operation calculates the value1 the value2 will be returned and vice versa.
+NOR(value1, value2) | This does a logical 'nor' operation. The value is 'calculated' by the normal 'or' operation and than negated by returning the opposite value. E.g. when the 'or' operation calculates the value1 the value2 will be returned and vice versa.
+OR(value1, value2)  | Does a logical 'or' operation. If at least one item is of 'value1' this is returned, otherwise the 'value2' is returned.
+SUM                 | Calculates the sum of all items in the group.
+
+An example of this would be:
+```xtend
+Group:Contact:OR(OPEN,CLOSED)  gMotionSensors  (All)
+```
+
+## Item names
```

Comment:
Fair point, i've moved it above the Groups section. It makes sense to have it after item types as that is the first part of an item string.